menpo/image/base.py

```diff
@@ -2257,6 +2258,101 @@ def rotate_ccw_about_centre(self, theta, degrees=True, retain_shape=False,
             shape, trans.pseudoinverse(), order=order, warp_landmarks=True,
             cval=cval, return_transform=return_transform)
 
+    def shear_about_centre(self, phi, psi, degrees=True, retain_shape=False,
```

Comment:
I think this is better (more general) as `transform_about_centre` where we just make sure that we apply the translation and then the translation back?